<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartUpDialog</class>
 <widget class="QDialog" name="StartUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StartUp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/client.png</normaloff>:/icons/icons/client.png</iconset>
  </property>
  <widget class="QLabel" name="lb_welcome">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_network">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Card: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_nickname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nick Name: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_network">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Switch Network Card</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Input Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Default: IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_connect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting for teacher...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Connect to Server</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Start Client</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
